### PR TITLE
use ubi as the base image for manager

### PR DESCRIPTION
**Pull request by ngoracke**
Signed-off-by: Nicholas Goracke <ngoracke@us.ibm.com>

### CODE REVIEW

File: manager/Dockerfile [hunk 1]
```diff
@@ -1,9 +1,10 @@
 # Copyright 2020 IBM Corp.
 # SPDX-License-Identifier: Apache-2.0
 
-FROM gcr.io/distroless/static:nonroot
-WORKDIR /
-COPY manager .
-USER nonroot:nonroot
+FROM ubi8/ubi-minimal
```
Comment (roee88) [inline manager/Dockerfile:4]: Should this be taken from registry.access.redhat.com?

File: manager/Dockerfile [hunk 2]
```diff
@@ -1,9 +1,10 @@
...
+ENV HOME=/tmp
+WORKDIR /tmp
```
Comment (roee88) [inline manager/Dockerfile:6]: Just wondering, is it a best practice to use /tmp as the workdir?

File: manager/Dockerfile (hunk 1 above)
Comment (ngoracke) [reply manager/Dockerfile:4]: I think that would be more explicit.  I'll change it

File: manager/Dockerfile (hunk 2 above)
Comment (ngoracke) [reply manager/Dockerfile:6]: This is something we've taken to doing for openshift.  Often, you'll be running with an SCC that randomizes the user ID you run as, so you want to be sure you've got access to execute the binary, and access to write any files to $HOME or PWD if the application requires it.

**Review by roee88 — APPROVED** [review]
We should replicate

**Pull request merged**